zerver/lib/push_notifications.py

```diff
@@ -817,7 +817,7 @@ def get_apns_alert_title(message: Message) -> str:
         return ", ".join(sorted(r["full_name"] for r in recipients))
     elif message.is_stream_message():
         return f"#{get_display_recipient(message.recipient)} > {message.topic_name()}"
-    # For personal PMs, we just show the sender name.
+    # For one-on-one direct messages, we just show the sender name.
```

Comment:
Elsewhere in this PR, I saw `1:1` being used.

Reply:
Good call. Switched to that format for this code comment.